web-animations-2/Overview.bs

```diff
@@ -969,6 +972,30 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  The [=animation/progress=] of an [=animation=] is calculated as follows:
```

Comment:
Should we introduce a variable `animation` here? i.e. something like:

```bs
The [=animation/progress=] of an [=animation=], |animation|, is calculated as follows:
```

Then make the later references to `animation` be `|animation|` and annotate the block with `algorithm` per https://speced.github.io/bikeshed/#var-and-algorithms ?